pennylane/interfaces/jax.py

```diff
@@ -472,6 +473,7 @@ def execute_wrapper_jvp(primals, tangents):
                         new_tapes,
                         tangents[0],
                         gradient_fn,
+                        device.shot_vector,
```

Comment:
Two points:
* This is a really neat solution, the user doesn't have to pass this in explicitly! :tada: 
* How about adding this into `gradient_kwargs` as a new entry: {"shots":device.shot_vector}? That way perhaps some signatures can stay as they are and `"shots"` could be obtained from `gradient_kwargs` via `gradient_kwargs.get("shots", None)`.

Reply:
Yes that was my initial solution, but I did not find it very clear. So I was thinking it was better to explicitly pass it. Also thinking about the upcoming changes and it will be attached to the tape. The readability is more clear like that in my opinion. But if you think it is much better with gradient_kwargs I can do the change 👍

Reply:
Sounds good, happy to keep this as is if the `gradient_kwargs` has been explored. :+1: